src/main/java/io/nats/client/Options.java

```diff
@@ -1337,19 +1337,21 @@ public Options build() throws IllegalStateException {
             
             if (servers.size() == 0) {
                 server(DEFAULT_URL);
-            } else if (servers.size() == 1) { // Allow some URI based configs
-                URI serverURI = servers.get(0);
-
-                if (TLS_PROTOCOL.equals(serverURI.getScheme()) && this.sslContext == null)
-                {
-                    try {
-                        this.sslContext = SSLContext.getDefault();
-                    } catch (NoSuchAlgorithmException e) {
-                        throw new IllegalStateException("Unable to create default SSL context", e);
+            }
+            else if (sslContext == null) {
+                for (URI serverURI : servers) {
+                    if (TLS_PROTOCOL.equals(serverURI.getScheme())) {

```

Comment:
Wondering if there should be a check someplace for a mix of TLS types - native vs openSSL should throw.  Could get into the situation where some URLS are openTLSContext and others are native ssl context.

Reply:
There is always the ability for the user to setup the sslContext themselves. I guess if this comes up we'll address it